tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -547,4 +547,32 @@ abstract class NodeJsActionContainerTests extends BasicActionRunnerTests with Ws
         e shouldBe empty
     })
   }
+
+  it should "allow running activations concurrently" in {
+    val (out, err) = withNodeJsContainer { c =>
+      val code =
+        """
+          | function main(args) {
+          |     console.log("interleave me");
+          |     return new Promise(function(resolve, reject) {
+          |       setTimeout(function() {
+          |         resolve({ done: true });
```

Comment:
i suggest changing the return to be the identity (return `args`) and passing in two different values to make sure the response is say either `{a: "A"}` or `{b: "B"}` and that both are received.